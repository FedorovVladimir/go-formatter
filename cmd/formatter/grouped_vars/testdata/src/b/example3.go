package main

var d = "a" // want "grouped vars"
var e = "b"
var f = "c"
