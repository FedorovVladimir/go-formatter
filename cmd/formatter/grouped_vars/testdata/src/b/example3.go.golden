package main

var (
	a = "a" // want "grouped vars"
	b = "b"
	c = "c"
)
